src/Helpers/IPFS.ts

```diff
@@ -0,0 +1,61 @@
+import * as FormData from 'form-data'
+import fetch, { Response } from 'node-fetch'
+import { prop } from 'ramda'
+import * as str from 'string-to-stream'
+
+import { minutesToMiliseconds } from './Time'
+
+const getHash = prop('Hash')
+
+interface IPFSConfiguration {
+  readonly url?: string
+}
+
+interface FetchOptions {
+  readonly timeout?: number
+}
+
+export const IPFS = ({
+  url = 'http://localhost:5001',
+}: IPFSConfiguration) => {
+  const paths = {
+    add: `${url}/api/v0/add`,
+    cat: `${url}/api/v0/cat`,
+  }
+
+  const addText = ({
```

Comment:
why `addText` and not `addContent`?

Reply:
This is old code, we are just moving it to a helper

Reply:
https://github.com/poetapp/node/blob/master/src/StorageWriter/IPFS.ts

Reply:
gotcha.